<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DashboardWindow</class>
 <widget class="QMainWindow" name="DashboardWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1420</width>
    <height>780</height>
   </rect>
  </property>
  <property name="palette">
   <palette>
    <active>
     <colorrole role="WindowText">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>255</red>
        <green>255</green>
        <blue>255</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Base">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>255</red>
        <green>255</green>
        <blue>255</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Window">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>36</red>
        <green>40</green>
        <blue>46</blue>
       </color>
      </brush>
     </colorrole>
    </active>
    <inactive>
     <colorrole role="WindowText">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>255</red>
        <green>255</green>
        <blue>255</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Base">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>255</red>
        <green>255</green>
        <blue>255</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Window">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>36</red>
        <green>40</green>
        <blue>46</blue>
       </color>
      </brush>
     </colorrole>
    </inactive>
    <disabled>
     <colorrole role="WindowText">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>120</red>
        <green>120</green>
        <blue>120</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Base">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>36</red>
        <green>40</green>
        <blue>46</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Window">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>36</red>
        <green>40</green>
        <blue>46</blue>
       </color>
      </brush>
     </colorrole>
    </disabled>
   </palette>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>1290</x>
      <y>100</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="palette">
     <palette>
      <active>
       <colorrole role="ButtonText">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>255</red>
          <green>255</green>
          <blue>255</blue>
         </color>
        </brush>
       </colorrole>
      </active>
      <inactive>
       <colorrole role="ButtonText">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>255</red>
          <green>255</green>
          <blue>255</blue>
         </color>
        </brush>
       </colorrole>
      </inactive>
      <disabled>
       <colorrole role="ButtonText">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>120</red>
          <green>120</green>
          <blue>120</blue>
         </color>
        </brush>
       </colorrole>
      </disabled>
     </palette>
    </property>
    <property name="font">
     <font>
      <family>Century Gothic</family>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Edit Info</string>
    </property>
    <property name="flat">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_2">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>92</y>
      <width>151</width>
      <height>41</height>
     </rect>
    </property>
    <property name="palette">
     <palette>
      <active>
       <colorrole role="ButtonText">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>255</red>
          <green>255</green>
          <blue>255</blue>
         </color>
        </brush>
       </colorrole>
      </active>
      <inactive>
       <colorrole role="ButtonText">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>255</red>
          <green>255</green>
          <blue>255</blue>
         </color>
        </brush>
       </colorrole>
      </inactive>
      <disabled>
       <colorrole role="ButtonText">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>120</red>
          <green>120</green>
          <blue>120</blue>
         </color>
        </brush>
       </colorrole>
      </disabled>
     </palette>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="flat">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="UserInterface_LeftSide_Clean_LBL">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>-10</y>
      <width>151</width>
      <height>791</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>../../../../Desktop/CS1D/Project Application/User Interface/LeftSidePNG_UserInterface/LeftSide_User.png</pixmap>
    </property>
   </widget>
   <widget class="QLabel" name="UserInterface_Hotbar_Clean_LBL">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>0</y>
      <width>1271</width>
      <height>91</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>../../../../Desktop/CS1D/Project Application/User Interface/HotbarPNG_UserInterface/HotbarNonuserInterface.png</pixmap>
    </property>
   </widget>
   <widget class="QLabel" name="UserInterface_Setting_LBL">
    <property name="geometry">
     <rect>
      <x>1280</x>
      <y>80</y>
      <width>141</width>
      <height>111</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>../../../../Desktop/CS1D/Project Application/User Interface/HotbarPNG_UserInterface/Emptyspacesauce.png</pixmap>
    </property>
   </widget>
   <widget class="QPushButton" name="UserInterface_Settings_BTN">
    <property name="geometry">
     <rect>
      <x>1280</x>
      <y>50</y>
      <width>141</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="flat">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="UserInterface_News_BTN">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>137</y>
      <width>151</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="flat">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QWebView" name="UserInterface_QWebView">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>90</y>
      <width>1271</width>
      <height>671</height>
     </rect>
    </property>
    <property name="url">
     <url>
      <string>about:blank</string>
     </url>
    </property>
   </widget>
   <widget class="QPushButton" name="UserInterface_Map_BTN">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>190</y>
      <width>151</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="flat">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="UserInterface_Weather_BTN">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>230</y>
      <width>151</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="flat">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="UserInterface_Trip_BTN">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>280</y>
      <width>151</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="flat">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="UserInterface_Shop_BTN">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>327</y>
      <width>151</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="flat">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="UserInterface_Tutorial_BTN">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>370</y>
      <width>151</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="flat">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="UserInterface_ShoppingCart_LBL">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>330</y>
      <width>141</width>
      <height>81</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>../../../../Desktop/CS1D/Project Application/User Interface/HotbarPNG_UserInterface/Emptyspacesauce.png</pixmap>
    </property>
   </widget>
   <widget class="QPushButton" name="UserInterface_SouveneerList_BTN">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>340</y>
      <width>111</width>
      <height>28</height>
     </rect>
    </property>
    <property name="palette">
     <palette>
      <active>
       <colorrole role="ButtonText">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>255</red>
          <green>255</green>
          <blue>255</blue>
         </color>
        </brush>
       </colorrole>
      </active>
      <inactive>
       <colorrole role="ButtonText">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>255</red>
          <green>255</green>
          <blue>255</blue>
         </color>
        </brush>
       </colorrole>
      </inactive>
      <disabled>
       <colorrole role="ButtonText">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>120</red>
          <green>120</green>
          <blue>120</blue>
         </color>
        </brush>
       </colorrole>
      </disabled>
     </palette>
    </property>
    <property name="font">
     <font>
      <family>Century Gothic</family>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Souvenir List</string>
    </property>
    <property name="flat">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="UserInterface_ShoppingCart_BTN">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>370</y>
      <width>121</width>
      <height>28</height>
     </rect>
    </property>
    <property name="palette">
     <palette>
      <active>
       <colorrole role="ButtonText">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>255</red>
          <green>255</green>
          <blue>255</blue>
         </color>
        </brush>
       </colorrole>
      </active>
      <inactive>
       <colorrole role="ButtonText">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>255</red>
          <green>255</green>
          <blue>255</blue>
         </color>
        </brush>
       </colorrole>
      </inactive>
      <disabled>
       <colorrole role="ButtonText">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>120</red>
          <green>120</green>
          <blue>120</blue>
         </color>
        </brush>
       </colorrole>
      </disabled>
     </palette>
    </property>
    <property name="font">
     <font>
      <family>Century Gothic</family>
      <pointsize>9</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Shopping Cart</string>
    </property>
    <property name="flat">
     <bool>true</bool>
    </property>
   </widget>
   <zorder>UserInterface_LeftSide_Clean_LBL</zorder>
   <zorder>UserInterface_Hotbar_Clean_LBL</zorder>
   <zorder>pushButton_2</zorder>
   <zorder>UserInterface_Setting_LBL</zorder>
   <zorder>pushButton</zorder>
   <zorder>UserInterface_Settings_BTN</zorder>
   <zorder>UserInterface_News_BTN</zorder>
   <zorder>UserInterface_QWebView</zorder>
   <zorder>UserInterface_Map_BTN</zorder>
   <zorder>UserInterface_Weather_BTN</zorder>
   <zorder>UserInterface_Trip_BTN</zorder>
   <zorder>UserInterface_Shop_BTN</zorder>
   <zorder>UserInterface_Tutorial_BTN</zorder>
   <zorder>UserInterface_ShoppingCart_LBL</zorder>
   <zorder>UserInterface_SouveneerList_BTN</zorder>
   <zorder>UserInterface_ShoppingCart_BTN</zorder>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QWebView</class>
   <extends>QWidget</extends>
   <header>QtWebKitWidgets/QWebView</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
